### ISSUE TITLE
Calbire-Web - Kindle friendly browser interface

### ISSUE THREAD

**Issue by IdefixRC** [janeczku/calibre-web#2723]
**Is your feature request related to a problem? Please describe.**
I currently use calibre and the included calibre content server on a windows machine to allow the various kindle devices in my house to directly download ebooks using the kindle web browser and access the non java page of the calibre server (/mobile).
The formatting of the mobile page works great for the kindle.
I'm now experimenting with calibre-web running in a docker container. it works great but accessing content via the kindle browser is way more troublesome than calibre server as the GUI of calibre-web does not format well on the kindle (way to small fonts, full calibre-web function etc) making downloads a bit cumbersome.

**Describe the solution you'd like**
I might be missing something but it would be great to have a calibre server (mobile version) type of interface option for calibre web to allow easier content download utilizing the kindle browser.

**Describe alternatives you've considered**
There is a calibre docker container which runs the standard calibre SW but it only works with a number of workarounds (e.g. the entire server needs to be restarted each time new books are added from remote calibre instances.

Thanks !

**Comment by IdefixRC** [comment]
Update: went with COPS (https://github.com/seblucas/cops) for now as I could not find a simple web interface for the kindle through calibre-web. Would be great to see something similar in calibre-web in the future.

**Comment by pnatel** [comment]
+1 on a kindle friendly version. 
Maybe just add the option to request the mobile version or the web version.
or add a /mobile to the site like Calibre.

**Comment by fferraro87** [comment]
+1

**Comment by carderne** [comment]
Yes I can experiment with that, a small-ish image should be no problem.
I can also add a tiny bit of CSS back just so it looks a bit nicer, but still optimised for the screen size etc.

Will update here when I get to it.

**Comment by carderne** [comment]
@pnatel 
Have updated my fork a bit:
1. Improved CSS
2. Added images
3. Added pagination

Give it a go by running the new Docker image [carderne/calibre-web:0.6.21-2](https://hub.docker.com/layers/carderne/calibre-web/0.6.21-2/images/sha256-57ca32b4ea4cf4802ceaf9cf7bb4a7dc21e53c2285074134dc489707423264a1?context=explore).

Or pull my repo [carderne/calibre-web](https://github.com/carderne/calibre-web) and do it manually.

**Comment by pnatel** [comment]
Thanks, @carderne, 
it was quite an improvement from 0.0.1. I added a few pics from the device for reference. The search button and the Home link appear to have swapped CSS, also the search button looks underneath the text field when it should be on its right side. 
Lastly, the pic could benefit from a resize, or maybe a tap to expand option. 
None of that prevents its usage, and the wife likes it too, so double thumbs-up!
I hope the feedback is useful.
## home page
![20240405_050919](https://github.com/janeczku/calibre-web/assets/52678829/473154e8-3abd-4caf-873e-7bdeca94d9db)
## tap to search
![20240405_051029](https://github.com/janeczku/calibre-web/assets/52678829/467725f4-9d9a-400f-b3cc-6312a811d508)
## Book details
![image](https://github.com/janeczku/calibre-web/assets/52678829/fafcef8b-ceb2-422b-8992-a09d824ebda1)

**Comment by carderne** [comment]
Just mentioning here that these issues were resolved in `0.6.21-3` as @pnatel clarified in this comment on the PR:
https://github.com/janeczku/calibre-web/pull/3008#issuecomment-2038188342

**Comment by carderne** [comment]
This has now been merged into main, hopefully released as a Docker image shortly
https://github.com/janeczku/calibre-web/pull/3008#issuecomment-2495503818
